ms2rescore/retention_time.py

```diff
@@ -0,0 +1,175 @@
+"""Add retention time related features to rescoring."""
+
+import logging
+from typing import Union
+
+import click
+import pandas as pd
+from deeplc import DeepLC
+
+from ms2rescore.peptide_record import PeptideRecord
+
+
+class RetentionTimeIntegration:
+    """Retention time integration for MS²ReScore, using DeepLC."""
+
+    def __init__(
+        self,
+        peprec_path: str,
+        feature_path: str,
+        higher_psm_score_better: bool = True,
+        num_calibration_psms: int = 500,
+        num_cpu: Union[int, None] = None,
```

Comment:
`Optional[int]` is more readable.

Reply:
Cool! Didn't know about this notation. I haven't really found a good resource on type hinting yet...

Reply:
The PEP might be a good one: https://www.python.org/dev/peps/pep-0484/